.circleci/config.yml

```diff
@@ -301,6 +301,7 @@ workflows:
                 - /^release-.*/
                 - /^foundation.*/
                 - /^features.*/
+                - /^mvr.*/
```

Comment:
Do you want to keep these here?

Reply:
Yes for now I'd like to keep these. I usually remove any statements which are not needed anymore after merging.